environment/image_recipes/ami/ami.json

```diff
@@ -54,9 +54,13 @@
       ],
 
       "tags": {
-        "Name": "{{user `image_name`}}",
+        "Name": "{{user `name`}}",
+        "Distribution": "{{user `distribution`}}",
+        "Release Label": "{{user `release_label`}}",
         "Build Date": "{{user `build_date`}}",
-        "Comment": "{{user `image_description`}}"
+        "Comment": "{{user `image_description`}}",
+        "Bundle Track": "{{user `bundle_track` }}",
```

Comment:
I think we've effectively ditched 'tracks' at the moment, but if the data is available we can still put it in.